database/migrations/2015_10_05_110622_create_conversations_table.php

```diff
@@ -1,20 +1,21 @@
 <?php
 
-use Illuminate\Support\Facades\Schema;
 use Illuminate\Database\Migrations\Migration;
-Use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
 
 class CreateConversationsTable extends Migration
 {
     /**
      * Run the migrations.
-     * 
+     *
      * @return void
      */
     public function up()
     {
         Schema::create('conversations', function (Blueprint $table) {
             $table->increments('id');
+            $table->varchar('title');
```

Comment:
Please make a new migration for this changes. otherwise old Talk users can not use this feature.

Reply:
This means this library will now depend on `doctrine/dbal` and so  `composer.json` must now have this dependency stated, thus old users will need to run `composer update`. My thoughts.

Reply:
Yep, If old users want to upgrade new version then he'll to be run migrate for getting new changes. otherwise users unable to upgrade these changes if we did't make separate migration.